miso-dto/src/test/java/uk/ac/bbsrc/tgac/miso/dto/DtosTestSuite.java

```diff
@@ -15,7 +20,7 @@
 public class DtosTestSuite {
```

Comment:
Please add test for converting to aliquot with new parent (stock, tissue, identity) info